README.md

```diff
@@ -40,6 +40,8 @@ optional arguments:
 API Options:
   -H HOSTNAME, --hostname HOSTNAME
                         OPNsense hostname or ip address
+  -H PORT, --port PORT  OPNsense https-api port
```

Comment:
Can you please fix the first parameter? Should be `-p`

Reply:
Thanks for the consideration. As far as I can see, you already fixed it - what should I do now?

Reply:
Sorry for the confusion, I'll merge it.